packages/popover/src/usePopover/usePopover.js

```diff
@@ -27,9 +38,15 @@ function usePopover({ openOnFocus, openOnHover, openOnMount, placement, onClose,
     !open && onClose && onClose()
   }
 
-  const handleToggle = (evt) => {
+  const handleToggle = (e) => {
     setOpen((isOpen) => {
-      isOpen ? onClose?.(evt) : onOpen?.(evt)
+      if (isOpen) {
+        onBeforeClose?.()
```

Comment:
Hey, I may not have the context here, but why is it that  `onBeforeClose?.()` and  `onBeforeOpen?.()` do not pass the `e`?

Reply:
Hey @DerrickTheodore, thanks for checking this out.  Just added the `e` back to the `onBeforeClose` and `onBeforeOpen`.